servatrice/migrations/servatrice_0008_to_0009.sql

```diff
@@ -0,0 +1,6 @@
+-- Servatrice db migration from version 8 to version 9
+
+alter table cockatrice_rooms add chat_history_size int(4) not null after join_message;
+update cockatrice_rooms set chat_history_size = 100;
```

Comment:
:+1: 

Can you add a message about this in the commit?

We need to notify server admins of the new configuration possibility